### PR TITLE
Add saved and delete favorite recipe functionality

### PR DESCRIPTION
**Pull request by forsethnico**
What’s this PR do?
- Added delete Favorite and save favorite functionality and allowed it to be displayed on the DOM

Where should the reviewer start?
- scripts.js line 240

How to test?
- pull down PR and input npm start in terminal and open [http://http://localhost:8080/](http://http//localhost:8080/) in browser

To do... still needs additional styling to space out the favorited recipe icons on the Favorites page.

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -219,16 +224,33 @@ function filterFavoriteRecipes(event) {
...
+  favoriteRecipeImages.innerHTML = '';
+  user.recipesToCook.forEach(recipe => {
+  favoriteRecipeImages.innerHTML += `<section class = "favorite-recipe-icon" >
+  <img class = "recipe-icons" src = ${recipe.image} id = ${recipe.id}>
+  </section>`;
+  });
+}
+
+function removeFromFavorites(event) {
+  let recipe = event.target
```
Comment (GOECHA) [inline src/scripts.js:251]: Great use of event.target to remove image and from array.

File: src/styles.css [hunk 2]
```diff
@@ -516,7 +517,7 @@ filter: drop-shadow(5px 10px 3px black);
        justify-content: center;
        align-items: center;
        height: 10%;
-       width: 25%; 
+       width: 25%;
        margin: 0;
```
Comment (GOECHA) [inline src/styles.css:521]: Need left margin to create space between images. Possibly adjust in the images section as well.

**Review by GOECHA — APPROVED** [review]
Great job removing recipes from the user.

File: src/scripts.js [hunk 3]
```diff
@@ -219,16 +224,33 @@ function filterFavoriteRecipes(event) {
...
+  showFavoriteRecipeImages()
+}
+
+function showFavoriteRecipeImages(){
+  favoriteRecipeImages.innerHTML = '';
+  user.recipesToCook.forEach(recipe => {
+  favoriteRecipeImages.innerHTML += `<section class = "favorite-recipe-icon" >
+  <img class = "recipe-icons" src = ${recipe.image} id = ${recipe.id}>
+  </section>`;
+  });
```
Comment (tfettig22) [inline src/scripts.js:247]: Nice!

**Review by tfettig22 — APPROVED** [review]

**Pull request merged**